README.md

```diff
@@ -1,17 +1,51 @@
+The Operations Engineering team buy, build, and run tools to help build and operate software at scale. While we primarily use the [Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) for developing and hosting our services, our goal is to create tools and standards that can be used across multiple platforms and hosting services throughout the organisation.
 
-This repository is published via Github Pages [here](https://ministryofjustice.github.io/operations-engineering/#moj-operations-engineering)
+## What's in This Repo
 
-To update, edit the files in the `source` directory. To run the site locally whilst editing use `make preview` and open `http://127.0.0.1:4567` within a browser. Use `make check` to compile the site to html and check the URLs are valid.
+This mono repository includes code that performs a variety of operations engineering tasks to streamline our workflow and maintain high operational standards. Here are some highlights:
 
-The config/tech-docs.yml file contains layout configuration options see [link](https://tdt-documentation.london.cloudapps.digital/configure_project/global_configuration/) for more details.
+- **Repository Reports:** We generate reports that verify the adherence of the Ministry of Justice organisation's repositories to the high standards outlined in our [Repository Standards](https://operations-engineering.service.justice.gov.uk/documentation/services/repository-standards.html). These reports help us maintain the quality of our code and streamline collaboration.
+
+- **Sentry Monitoring:** We monitor Sentry for over and under-utilisation, ensuring we're leveraging this error tracking tool to its full potential. This helps in identifying and rectifying application errors more efficiently.
+
+- **Dormant User Detection:** Our code can detect inactive GitHub users and remove them from the organisation, keeping our workspace tidy and secure.
+
+These are just a few examples of the useful tools and features available in this repository. For more detailed information about each tool, feature, and how they assist us in our operations, see the GitHub workflows in the `.github/workflows` directory for more information.
+
+## Getting Started
+
+1. **Clone the Repo:** `git clone https://github.com/operations-engineering.git`
+2. **Navigate to the Repo:** `cd operations-engineering`
+3. **Install Dependencies:** `pip install -r python/requirements.txt`
+4. **Run a script:** `python python/scripts/<example>.py`
+
+# Documentation Hosting and Updates
+
+Our repository documentation is built using the [Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/) to provide a user-friendly and consistent experience. The Technical Documentation Template ensures a clear, standardised layout for all our project documentation, enhancing readability and navigation.
```

Comment:
I would recommend this url: https://github.com/ministryofjustice/template-documentation-site

Reply:
or https://github.com/ministryofjustice/tech-docs-github-pages-publisher